chromium_src/content/public/browser/browser_context.h

```diff
@@ -6,16 +6,17 @@
 #ifndef BRAVE_CHROMIUM_SRC_CONTENT_PUBLIC_BROWSER_BROWSER_CONTEXT_H_
 #define BRAVE_CHROMIUM_SRC_CONTENT_PUBLIC_BROWSER_BROWSER_CONTEXT_H_
 
-#include "../../../../../content/public/browser/browser_context.h"
-
 #include <string>
 
 #include "base/memory/ref_counted.h"
+#include "brave/content/browser/ephemeral_storage_partition.h"
```

Comment:
This is an issue because we are including a non-public header. I will fix this in a new version of the branch.